src/tap.h

```diff
@@ -32,9 +32,38 @@
 #include <iostream>
 #include <map>
 #include <fstream>
+#include <string>
 
 namespace tap {
 
+// based on http://stackoverflow.com/a/7724536/831180
```

Comment:
As they weren't any, I though that I could be lazy :stuck_out_tongue: 

I'll do.